docs/reference/content/reference/unified-topology/index.md

```diff
@@ -0,0 +1,78 @@
++++
+date = "2019-07-04T12:53:30-04:00"
+title = "Unified Topology Design"
+[menu.main]
+  parent = "Reference"
+  identifier = "Unified Topology Design"
+  weight = 100
+  pre = "<i class='fa'></i>"
++++
+
+# Unified Topology Design
+
+At the time of writing the node driver has seven topology classes, including the newly introduced unified topology. Each legacy topology type from the core module targets a supported topology class: Replica Sets, Sharded Deployments (mongos) and Standalone servers. On top of each of these rests a thin topology wrapper from the "native" layer which introduces the concept of a "disconnect handler", essentially a callback queue for handling naive retryability.
+
+The goal of the unified topology is threefold:
+  - fully support the drivers Server Discovery and Monitoring, Server Selection and Max Staleness specifications
+  - reduce the maintenance burden of supporting the topology layer in the driver by modeling all supported topology types with a single engine
+  - remove confusing functionality which could be potentially dangerous for our users
+
+## How to use it
+
+The unified topology is available now behind the `useUnifiedTopology` feature flag. You can opt in to using it by passing the option to your `MongoClient` constructor:
+
+```js
+const client = MongoClient('mongodb://localhost:27017', { useUnifiedTopology: true });
+```
+
```

Comment:
We should add:

1. Eventually, in later minor versions, `useUnifiedTopology` will default to `true`
2. In the next major version, the unified topology will be the default, and we will remove the legacy topology behavior.

Reply:
added a sentence for this